src/app/tabs/compile-tab.js

```diff
@@ -305,17 +305,17 @@ function compileTab (container, appAPI, appEvents, opts) {
       }
       if (!error) {
         if (data.contracts) {
-          for (var contract in data.contracts) {
-            appAPI.compilationMessage(contract, $(errorContainer), {type: 'success'})
-          }
+          appAPI.visitContracts((contrract) => {
+            appAPI.compilationMessage({ formattedMessage: contrract.name, severity: 'success' }, $(errorContainer))
```

Comment:
There is no `severity: success`. What is this code doing?

Reply:
ah yes I meant `type` not `severity`